File: begginers/average_3.cpp
#include<iostream>

using namespace std;

int main(){
  float n1, n2, n3, n4, n5, average;

  cin >> n1 >> n2 >> n3 >> n4;

  average = (n1*2 + n2*3 + n3*4 + n4)/10;

  cout.precision(2);

  cout << "Media: " << average << endl;

  if (average >= 7.0){
    cout << "Aluno aprovado.\n";
  } else if(average < 5.0){
    cout << "Aluno reprovado.\n";
  } else if (average >= 5 && average < 7){
    cout << "Aluno em exame.\n";
    cin >> n5;

    cout << "Nota do exame: " << n5 << endl;

    average = (average + n5)/2;

    if (average >= 5){
      cout << "Aluno aprovado.\n";
    }else if(average < 5){
      cout << "Aluno reprovado.\n";
    }

    cout << "Media final: " << average << endl;
  }

  return 0;
}

File: begginers/average_3.py
line = raw_input().split(" ")
n1 = float(line[0])
n2 = float(line[1])
n3 = float(line[2])
n4 = float(line[3])

average = (n1*2 + n2*3 + n3*4 + n4)/10;

print "Media: %.1f" % average

if (average >= 7.0):
    print "Aluno aprovado."
elif(average < 5.0):
    print "Aluno reprovado."
elif (average >= 5 and average < 7):
    print "Aluno em exame."
    n5 = float(raw_input())

    print "Nota do exame: %.1f" % n5
    
    average = (average + n5)/2

    if (average >= 5):
        print "Aluno aprovado."
    elif(average < 5):
        print "Aluno reprovado."

    print "Media final: %.1f" % average
